src/Controller/MoocController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use AppBundle\Entity\Mooc\AttachmentFile;
+use League\Flysystem\FilesystemInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\ResponseHeaderBag;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration as SFConf;
+
+/**
+ * @SFConf\Route("/mooc")
+ */
+class MoocController extends Controller
+{
+    /**
+     * @SFConf\Route("/file/{id}", requirements={"id": "\d+"}, name="mooc_get_file")
+     * @SFConf\Method("GET")
+     * @SFConf\Cache(maxage=900, smaxage=900)
+     */
+    public function getFile(FilesystemInterface $filesystem, AttachmentFile $file): Response
```

Comment:
Does it require some kind of special permission to access these files ? Be logged in ? Be the owner ? Be part of the mooc ?

Reply:
It will be attach to each lesson, as I know there are no permission no login from users to get quizz lessons, files , ....

Reply:
nice thank you :-)